sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -246,18 +248,31 @@ object Webpack {
     library
   }
 
-  private def jsonOutput(logger: Logger)(in: InputStream): Option[WebpackStats] = {
-
-    val parsed = Json.parse(in)
-    parsed.validate[WebpackStats] match {
-      case JsError(e) =>
-        logger.error("Error parsing webpack stats output")
-        // In case of error print the result and return None. it will be ignored upstream
-        e.foreach {
-          case (p, v) => logger.error(s"$p: ${v.mkString(",")}")
-        }
+  private def jsonOutput(cmd: Seq[String], logger: Logger)(in: InputStream): Option[WebpackStats] = {
+    Try {
+      val parsed = Json.parse(in)
```

Comment:
I probably didn't understand `play-json` well enough. `Json.parse` can throw and that leaves the system in an unstable state